tests/v2/validation/charts/resources/alertmanagerstypes.go

```diff
@@ -0,0 +1,440 @@
+package resources
+
+import (
```

Comment:
@caliskanugur shouldn't these structs go in the framework instead? Just asking to make sure, I'm suggesting anything

Reply:
I see. Thank you .